dist/loadjs.js

```diff
@@ -108,17 +108,13 @@ function loadScripts(paths, callbackFn) {
   var i = paths.length, numWaiting = i, pathsNotFound = [], fn;
   
   // define callback function
-  fn = function(path, result, ev) {
-    // handle error
-    if (result[0] == 'e') pathsNotFound.push(path);
-
-    // handle beforeload event. If defaultPrevented then that means the load will be blocked (ex. Ghostery on Safari)
-    else if (result[0] == 'b') {
-        if (ev.defaultPrevented) pathsNotFound.push(path);
-        else return;
-    }
+  fn = function(path, result, defaultPrevented) {
+    // handle error and defaultPrevented (e.g. from ad blockers)
+    if (result == 'e' || defaultPrevented) pathsNotFound.push(path);
     
-    numWaiting--;
+    // decrement counter if result type is not 'beforeload'
+    if (result != 'b') numWaiting--;
```

Comment:
Wouldn't we want to decrement the counter if (result == 'b' && defaultPrevented)? If we detect that condition then we assume that there will be no error or load callback and so should numWaiting-- so that the callbackFn may be called.